packages/react-aria-components/stories/Autocomplete.stories.tsx

```diff
@@ -751,9 +779,11 @@ export function AutocompleteWithExtraButtons() {
   );
 }
 
+// TODO: note that Space is used to select an item in a multiselect menu but that is also reserved for the
+// autocomplete input field. Should we add logic to allow Space to select menu items when focus is in the Menu
+// or is that a rare/unlikely use case for menus in general?
 export const AutocompleteMenuInPopoverDialogTrigger = {
```

Comment:
Something I noticed when implementing this change. We could possibly have the menu handle the `Space` even when focus is in the menu but would that be confusing to the user?

Reply:
I think this would be confusing. if I'm typing in a textfield, that is my intent, not selection with possible repercussions like closing on selection.

Reply:
do we remember why Enter doesn't select stuff in menus but does everywhere else?

Reply:
Doesn't Enter select in Menu? https://reactspectrum.blob.core.windows.net/reactspectrum/3c9b36ae9e9bb42792ac4b3a51121969ae8d579c/storybook/index.html?path=/story/menutrigger--close-on-select-true-multiselect-menu&providerSwitcher-express=false
I thought it was that there was a difference in closeOnSelect where keyboard users could choose which behaviour they wanted regardless of the prop being set true

Reply:
The aria spec also distinguishes between Enter and Space: https://www.w3.org/WAI/ARIA/apg/patterns/menubar/. It states that Enter should always activate the item and close the menu unless it is a submenu trigger